### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by RaczeQ**

### CODE REVIEW

**Comment by RaczeQ** [comment]
Closes #91, #68

**Comment by RaczeQ** [comment]
Closes #68

**Comment by RaczeQ** [comment]
Closes #91

**Comment by RaczeQ** [comment]
Closes #69

File: docs/assets/css/palette.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:root > * {
+  --srai-brand-color: #01fdb1;
```
Comment (Calychas) [inline docs/assets/css/palette.css:2]: TBH, I feel a lot of eyestrain looking at this color in the docs. Maybe a little bit darker color like `#00eda5`?

File: mkdocs.yml [hunk 2]
```diff
@@ -15,25 +20,25 @@ theme:
...
-    logo: material/earth
     repo: material/github
   palette:
     # Palette toggle for light mode
     - media: "(prefers-color-scheme: light)"
       scheme: default
-      primary: orange
-      accent: light blue
+      primary: white
+      accent: deep orange
```
Comment (Calychas) [inline mkdocs.yml:32]: probably should change from orange to `#150b5b` ?

File: mkdocs.yml [hunk 3]
```diff
@@ -15,25 +20,25 @@ theme:
...
         icon: material/brightness-7
         name: Switch to dark mode
 
     # Palette toggle for dark mode
     - media: "(prefers-color-scheme: dark)"
       scheme: slate
-      primary: orange
-      accent: light blue
+      primary: white
+      accent: orange
```
Comment (Calychas) [inline mkdocs.yml:41]: Same here

**Review by Calychas — CHANGES_REQUESTED** [review]
Great work

**Review by Calychas — APPROVED** [review]

**Pull request merged**